src/datreant.core.egg-info/PKG-INFO

```diff
@@ -0,0 +1,83 @@
+Metadata-Version: 1.1
```

Comment:
https://help.github.com/articles/removing-sensitive-data-from-a-repository/

This helps tells you all you need to know. I used the filter branch script several times myself or to help out colleagues.  You can test it out on a new branch but it should work fine